src/swsssdk/scripts/sonic-db-cli

```diff
@@ -36,3 +47,35 @@ else:
             print "\n".join(resp)
         else:
             print resp
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                                     formatter_class=argparse.RawTextHelpFormatter,
+                                     epilog=
+"""
+Example 1: sonic-db-cli CONFIG_DB keys *
+Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
+Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
+Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+Example 5: sonic-db-cli PING
+""")
+    parser.add_argument('db_or_op', type=str, help='Database name Or Unary operation(only PING supported)')
+    parser.add_argument('cmd', nargs='*', type=str, help='Command to execute in database')
+
+    args = parser.parse_args()
+
+    if args.db_or_op:
+        if args.cmd:
+            execute_cmd(args.db_or_op, args.cmd)
+        elif args.db_or_op == 'PING':
+            print ping_all_instances()
+#       TODO next PR will support 'SAVE' and 'FLUSHALL'
+#       elif args.db_or_op == 'SAVE':
+#       elif args.db_or_op == 'FLUSHALL':
+        else:
+            parser.print_help()
+    else:
+        parser.print_help()
+
+if __name__ == "__main__":
```

Comment:
Should we use different return code to indicate different cases?

Reply:
This cmd is to mimic redis-cli, return codes I think is not necessary, it already has output msg.

Reply:
return codes handled